librz/core/cmd_flag.c

```diff
@@ -1216,7 +1216,35 @@ RZ_IPI int rz_cmd_flag(void *data, const char *input) {
 			}
 			break;
 		case 's':
-			flag_space_stack_list(core->flags, input[2]);
+			RzOutputMode mode;
```

Comment:
You could extract this in a separate function

Reply:
@XVilka I just wanted to confirm one thing. Should I create an `RZ_API` defined function in this file that I can then call in every other file as well?